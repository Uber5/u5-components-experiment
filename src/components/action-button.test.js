/* eslint-disable */
import React from 'react'
import renderer from 'react-test-renderer'
import ActionButton from './action-button'

describe('action button', () => {
  it('renders', () => {
    const component = renderer.create(<ActionButton onClick={() => console.log('clicked')}>do this</ActionButton>)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
