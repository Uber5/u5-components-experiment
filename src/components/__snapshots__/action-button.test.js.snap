// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`action button renders 1`] = `
<button
  className="md-btn md-btn--raised md-btn--text md-pointer--hover md-background--primary md-background--primary-hover md-inline-block"
  disabled={undefined}
  href={undefined}
  onClick={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchStart={[Function]}
  type="button"
>
  <div
    className="md-ink-container"
    style={undefined}
  />
  do this
</button>
`;
